src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -816,6 +854,36 @@ private void RunTestAsSudo(Func<string, int> testMethod, string arg)
         [DllImport("libc")]
         private static extern uint geteuid();
 
+        [DllImport("libc")]
+        private static extern uint getuid();
+
+        [DllImport("libc")]
+        private static extern uint getgid();
+
+        [DllImport("libc", SetLastError = true)]
+        private unsafe static extern int getgroups(int size, uint* list);
+
+        private unsafe static uint[] GetGroups()
+        {
+            int maxSize = 128;
+            Span<uint> groups = stackalloc uint[maxSize];
+            fixed (uint* pGroups = groups)
```

Comment:
I could make groups a `uint*` but than later I can no longer use the `Slice` method. So I think it's needed because I'm using a `Span`.

Reply:
> So I think it's needed because I'm using a Span.

Right.  The language doesn't track that the span was created from a stackalloc, and thus doesn't see that fixed isn't actually needed.

Reply:
I'm sure it's a bad idea and/or already discussed but I opened https://github.com/dotnet/csharplang/issues/2249 anyway